data/pages.yml

```diff
@@ -200,6 +200,8 @@
       url: "configuring-your-app"
     - title: "Configuring Ember CLI"
       url: "configuring-ember-cli"
+    - title: "Preparing for Ember Upgrades"
+      url: "preparing-for-upgrade"
```

Comment:
Maybe the title should be something like "Handling Deprecations" or "Deprecations in Ember"